### PR TITLE
Handle "container" option of clipboardJS

### PR DESCRIPTION
**Pull request by enna**
This PR handles the `container` option of ClipboardJS, as described [here](https://github.com/zenorocha/clipboard.js/pull/368).This `container` option allows the clipboard to work in modals, which will fix issue #180

### CODE REVIEW

**Comment by enna** [comment]
@jkusa sure, no prob. :)

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ http://jkusa.github.io/ember-cli-clipboard
 - `clipboardText` - string value or action that returns a string to be copied
 - `clipboardTarget` - selector string of element from which to copy text
 - `clipboardAction` - string value of operation: `copy` or `cut` (default is copy)
+- `container` - selector string of element, or element, containing the component. If you want to use the component inside a modal, this property is mandatory and should referred to the modal element.
```
Comment (jkusa) [inline README.md:63]:
```suggestion
- `container` - selector string or element object of containing element. "For use in Bootstrap Modals or with any other library that changes the focus you'll want to set the focused element as the container value".
```

**Comment by jkusa** [comment]
@enna change look good. Just a small tweak to the readme and we should be good.

**Comment by enna** [comment]
All should be good now @jkusa :)

**Comment by drewctate** [comment]
We could really use this improvement! Any plan on cutting an NPM release with container support @jkusa?

**Pull request merged**